src/GetFunctionsMetadataCommand.cs

```diff
@@ -31,25 +32,39 @@ protected override void ProcessRecord()
         {
             try
             {
-                if (!string.IsNullOrEmpty(FunctionsAppDirectory) && Directory.Exists(FunctionsAppDirectory))
+                // Attempt to normalize the user's input to a local path from where the cmdlet is running
+                SessionState ss = new SessionState();
```

Comment:
Could you please add some test cases for this code path? Something like this:

```
Get-FunctionsMetadata $PSScriptRoot\TestAppOne

Get-FunctionsMetadata .\TestAppOne

Get-FunctionsMetadata .\..\TestAppOne
```

Reply:
Done